doc/man3/EVP_PKEY_CTX_ctrl.pod

```diff
@@ -161,6 +161,17 @@ up to the size of the DH prime B<p>.
 If B<OSSL_EXCHANGE_PARAM_PAD> is zero (the default) then no padding is
 performed.
 
+=item OSSL_SIGNATURE_PARAM_MD_NAME (UTF8 string type)
+
+Sets the name of the digest algorithm used for the input to the signature
```

Comment:
Regarding the item line, I think we should show the actual name (i.e. the name string).  I'm going to insist on it, actually.

Reply:
> Regarding the item line, I think we should show the actual name (i.e. the name string). I'm going to insist on it, actually.

Instead of, or as well as the define?

Reply:
If you ask me, I would prefer the string to be the primary thing, but I'm absolutely with having existing macros displayed along with it, just as long as we don't make it look like the macro is the *only* possibility

Reply:
The KDFs and MACs use the form:

```
=item B<OSSL_MAC_PARAM_DIGEST> ("digest") <utf8 string>
```

Reply:
There has been a suggestion to change this to:
```
=item  "digest" B<OSSL_MAC_PARAM_DIGEST> <utf8 string>
```

Reply:
Or `=item  "digest" (B<OSSL_MAC_PARAM_DIGEST>) <utf8 string>`
(i.e. parenthesis added)

I would prefer this, i.e. emphasis on the string.

Reply:
Is bare angled brackets around "utf8 string" correct pod syntax? It looks odd to me.

Reply:
It's perfectly acceptable pod syntax.  If you read [the perlpod manual](https://perldoc.perl.org/perlpod.html) carefully, you will see that the only real reason to use constructs like `E<lt>` and `E<gt>` is to disambiguate ambiguous cases, and that there are other ways that avoids that kind of construct.

(I actually asked myself the exact same question a few days ago, which is the reason I know this quite well now)

Reply:
I made the edits as suggested by both @paulidale and @levitte.